src/main/java/de/ait/secondlife/controllers/OfferController.java

```diff
@@ -63,14 +63,31 @@ public ResponseEntity<OfferResponseWithPaginationDto> getAll(
                     @ExampleObject(name = "Sort by title", value = "title"),
                     @ExampleObject(name = "Sort by start price", value = "startPrice")
             })
-            String sortBy) {
-        Pageable pageable;
-        try {
-            pageable = PageRequest.of(page, size, Sort.by(sortBy).ascending());
-        } catch (IllegalArgumentException e) {
-            throw new PaginationParameterIsWrongException(page, size, sortBy);
-        }
-        return ResponseEntity.ok(service.findOffers(pageable));
+            String sortBy,
+            @RequestParam(defaultValue = "true")
+            @Parameter(description = "Sorting direction.", examples = {
+                    @ExampleObject(name = "Sort direction is ascending(default)", value = "true"),
+                    @ExampleObject(name = "Sort direction is descending", value = "false")
+            })
+            Boolean isAsc,
+            @RequestParam(required = false)
+            @Parameter(description = "Category id for filtration. Can be null." +
+                    " Optional parameter", example = "3")
+            Long category_id,
+            @RequestParam(required = false)
+            @Parameter(description = "Offer status for filtration. Can be null." +
+                    " Optional parameter", example = "DrAfT")
```

Comment:
in what style is the word _draft_ written? this is not snake_case, and not a CamelCase, but some kind of dRunKenCAse :)